### DISCUSSION TITLE
VSCode Please add an option disable the warning tooltips!

### DISCUSSION THREAD

**Discussion by ShravanSunder** [biomejs/biome-vscode#24] [General]
There shoudl be a way to disable warnings in the vscode extension.  I don't want to see if something i setup as an  explicit option fail:  
![2023-11-12 1130 Brave Browser New Discussion · biomejsbiome](https://github.com/biomejs/biome/assets/5294949/a191a702-8d82-494b-9b65-edfe90bd7078)

There's already a vscode output log if i'm really interested in these messages.  This is the standard for vscode extension.  Only user actionable messages should show up in the dialog.

This is extremely frustrating as it takes up screen real estate and blocks my console.

**Comment by ematipico** [comment]
Can you share some resources about the standard you mentioned?

I'm sure these messages haven't been put there to make your DX stressful.

Would you propose an alternative and make a PR if you think there's a better way to handle this? It seems you know a lot

**Comment by ShravanSunder** [comment]
hi @ematipico! thanks for the amazing biomejs!
Here's the notifications system link in vs-code https://code.visualstudio.com/api/ux-guidelines/notifications

Here are the relevant sections 
> It's important to limit the number of notifications sent in order to respect the user's attention. To help guide your decision on whether or not you should show a notification, please follow our notification decision tree:


I would posit that formatting warnings **WHILE** i'm editing a file or in the processes of doing something is not a "high priority error", or something the user needs to be alerted about.  Additionally, Having repeated notifications blocks the UX in that space especially if we have console, or debug windows there, in that case, they should be combined or have a debounce.  

![2023-11-30 1109 CleanShot CleanShot](https://github.com/biomejs/biome/assets/5294949/9f73375b-a96e-436e-99c2-07cfd4e9c02b)

**Comment by ematipico** [comment]
This will be addressed in https://github.com/biomejs/biome/pull/1134
